pom.xml

```diff
@@ -65,41 +65,41 @@
 
         <dependency>
             <groupId>org.apache.flink</groupId>
-            <artifactId>flink-jdbc_2.11</artifactId>
-            <version>${flink.version}</version>
+            <artifactId>flink-jdbc_${scala.binary.version}</artifactId>

```

Comment:
Upgrading to Flink 1.19 should replace the obsolete Flink JDBC `org.apache.flink:flink-jdbc_2.12:1.10.3` with the new JDBC connector (`org.apache.flink:flink-connector-jdbc:3.2.0-1.19`) using the new API.